### ISSUE TITLE
duration has to be >= 60 seconds

### ISSUE THREAD

**Issue by jsanda** [thelastpickle/tlp-stress#109]
I discover this while writing an integration test for my k8s operator. My test runs tlp-stress with the following args:

```
tlp-stress run KeyValue --duration 20s
```
In the logs for my tlp-stress pod, I saw:

```
Executing 1000000 operations with consistency level LOCAL_ONE
```
I confirmed that this is because `HumanReadableTimeConverter.convert` returns `duration.toMinutes()`.

It is somewhat of a minor nit, but it might be worth documenting.

**Comment by rustyrazorblade** [comment]
Good catch.  We should either throw an error or allow seconds to be provided.  I don't personally see much use for seconds; stress testing for 10s doesn't tell you anything, but there might be some valid cases for it around debugging like you've using there.

**Comment by jsanda** [comment]
My use case was for an integration test in which I needed to verify tlp-stress ran. Outside of that I agree that there is not much using for a duration < 60 sec.
